dev-manual/api/api-reference-storage-service.rst

```diff
@@ -540,13 +544,330 @@ Where payload.json contains::
       "endpoint_url": "http://127.0.0.1:12345",
       "access_key_id": "_Cah4cae1_",
       "secret_access_key": "_Thu6Ahqu_",
-      "region": "us-west-2"
+      "region": "us-west-2",
+      "bucket": "bucket_name"
     }
 
 ..  note::
 
     Endpoint returns all paths base64 encoded.
 
+Access protocols
+^^^^^^^^^^^^^^^^
+
+Archipelago
+^^^^^^^^^^^
+
+Archipelago is currently supported for AIP Storage locations.
+Archipelago is built on the content management system Drupal,
+which accepts JSON API calls to create new entities.
+Basic fields for a local file system space are listed `above \
+<https://www.archivematica.org/en/docs/archivematica-1.15/dev-manual/api/api-reference-storage-service/#create-space>`_
```

Comment:
Should we make the links in the descriptions of each access protocol [internal](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#internal-links)? That way we don't need to update them on each release (notice how some of the existing links in this page still point to the Storage Service 0.18 docs).

If for some reason you still need to use full links to the docs, using `latest` in the version (e.g `archivematica-1.15`) could be more maintainable.